man/nvim.1

```diff
@@ -393,8 +393,6 @@ The system-global
 .Nm
 runtime directory.
 .El
-.Sh SEE ALSO
-.Xr nvimtutor 1
```

Comment:
In this file, make sure to also remove the argument to `.Os` (line 3): it's only purpose was to signify that `nvim` and `nvimtutor` were part of the "Neovim" distribution, so one and the same, which isn't applicable anymore because `nvimtutor.1` is being removed.

If you need clarification on why it should be kept without arguments, here's it referenced in `man mdoc`:
```
Os
     Operating system version for display in the page footer.  This is the
     mandatory third macro of any mdoc file.  Its syntax is as follows:

           .Os [system [version]]

     The optional system parameter specifies the relevant operating system or
     environment.  It is suggested to leave it unspecified, in which case
     mandoc(1) uses its -Ios argument or, if that isn't specified either,
     sysname and release as returned by uname(3).

     Examples:
           .Os
           .Os KTH/CSC/TCS
           .Os BSD 4.3

     See also Dd and Dt.

```